tests/test_online_replacing.py

```diff
@@ -0,0 +1,30 @@
+import unittest
+import os
+from requre.online_replacing import update_module
+
+from tests.data.special_requre_module import hello
+
+SELECTOR = str(os.path.basename(__file__).rsplit(".", 1)[0])
+
+def decorator_a(func):
+    def _int():
+        return "a" + func()
+    return _int
+
+def decorator_b(func):
+    def _int():
+        return "b" + func()
+    return _int
+
+class TestOnlinePatching(unittest.TestCase):
+
+    @update_module(module_name_filter=SELECTOR, selector="hello", replacement=decorator_a)
```

Comment:
We need to make names and attribute more self-descriptive.

Something like this?

```suggestion
    @replace(what="hello", in_module="special_requre_module", decorate_by=decorator_a)
```

Reply:
sure, I'll make it nicer when I make it workings as expected